src/models.py

```diff
@@ -81,6 +83,9 @@ def load(self, filepath):
             self.model = self.model.cuda()
         else:
             self.model.load_state_dict(torch.load(filepath, map_location=lambda storage, loc: storage))
+
+        self.model.train()
```

Comment:
@taraspiotr is this necessary. What if i simply load model to run eval? I guess I it is changed to eval mode in transform() but still this seems a bit odd.

Reply:
@jakubczakon but what if we load a model from checkpoint to train it? I've seen that we follow the convention, that model is always in train mode and when we need it we change it to eval and back to train afterward.

Reply:
@taraspiotr what is the default mode then. Isn't it train ?

Reply:
@taraspiotr ok good thanks